fluentbit.tf

```diff
@@ -11,15 +11,28 @@ locals {
     volumesFrom            = []
 
     environment = [
-      // Valid values are: debug, info and error
-      { name = " FLB_LOG_LEVEL", value = "error" }
+      // Valid values are: debug, info and error, default if missing: info
+      { name = "FLB_LOG_LEVEL", value = "error" },
+      {
+        "name" : "aws_fluent_bit_init_s3_1",
+        "value" : "arn:aws:s3:::config-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}/ecs/fluent-bit/service-custom.conf"
+      },
+      {
+        "name" : "aws_fluent_bit_init_s3_2",
+        "value" : "arn:aws:s3:::config-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}/ecs/fluent-bit/filters-custom.conf"
+      },
+      {
+        "name" : "aws_fluent_bit_init_s3_3",
+        "value" : "arn:aws:s3:::config-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}/ecs/fluent-bit/parsers-custom.conf"
+      },
```

Comment:
this seems very specific to our use-case, not sure if we set this as a default for all users.

what happens if those files are missing at startup of the service?

Reply:
The `init` tag causes the image to start with the `fluent-bit-init.conf` ...

The files passed to the `init` process will either be `@INCLUDE` in the config or if there are parsers added to the `CMD` via the `-R` parameter.

If there are no input to the `init` process, the config simply loads the default config:

```shell
$ cat /init/fluent-bit-init.conf
@INCLUDE /fluent-bit/etc/fluent-bit.conf
```